pycbc/workflow/grb_utils.py

```diff
@@ -575,3 +510,404 @@ def create_node(self, input_file, out_dir, ifo_tag, segment, tags=None):
                         os.path.join(out_dir, out_name), tags=tags)
         node.add_output_opt('--output-file', out_file)
         return node, out_file
+
+
+# [Never invoked outside this file]
+def opt_to_file(workflow, section, option):
```

Comment:
Consider adding this to core. You could use it elsewhere (see comment on bank_veto_bank, which can basically be replaced with "use this"). If adding to core, please do use a clearer name (e.g. configparser_value_to_file) to distinguish from command line options. Also add in the Inputs and Returns fields in the docstring as for other functions there.

Reply:
@spxiwh, I followed up on this and I am not marking it as resolved as it touches core. I.e., let's look at it carefully :-)